src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1727,15 +1765,17 @@ private void AggregateRenderPackages(IEnumerable<HelixRenderPackage> packages)
                     {
                         meshGeometry3D = CreateDynamoGeometryModel3D(rp);
                         Element3DDictionary.Add(id, meshGeometry3D);
+                        OnRequestAddViewportItem(meshGeometry3D);
                     }
 
-                    var mesh = meshGeometry3D.Geometry == null
-                        ? HelixRenderPackage.InitMeshGeometry()
+                    var mesh = meshGeometry3D.Geometry == null ? HelixRenderPackage.InitMeshGeometry() 
                         : meshGeometry3D.Geometry as MeshGeometry3D;
                     var idxCount = mesh.Positions.Count;
 
                     mesh.Positions.AddRange(m.Positions);
 
+                    //meshGeometry3D.Material = new VertColorMaterial();
```

Comment:
This was recommended by the Helix developer when needing to assign individual vertex colors however doing this, makes the geometry to disappear.